### PR TITLE
Add Grizzly and Undertow to Networking

### PR DESCRIPTION
**Pull request by yegorius**
Undertow is here https://github.com/undertow-io/undertow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -281,8 +281,10 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries for network programming.*
 
+* [Grizzly](https://grizzly.java.net/) - NIO framework. Used as a network layer in Jetty.
```
Comment (flowersinthesand) [inline README.md]: Isn't it Glassfish not Jetty?

Comment (yegorius) [inline README.md]: True, I confused Glassfish and Jetty

File: README.md [hunk 2]
```diff
@@ -281,8 +281,10 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries for network programming.*
 
+* [Grizzly](https://grizzly.java.net/) - NIO framework. Used as a network layer in Glassfish.
```
Comment (akullpp) [inline README.md]: Why in the Networking section and not in Server?

File: README.md [hunk 3]
```diff
@@ -281,8 +281,10 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
...
 * [Netty](http://netty.io/) - A framework for building high performance network applications.
 * [OkHttp](http://square.github.io/okhttp/) - An HTTP+SPDY client for Android and Java applications.
+* [Undertow](http://undertow.io/) - Web server providing both blocking and non-blocking API’s based on NIO. Used as a network layer in WildFly.
```
Comment (akullpp) [inline README.md]: What is a valid use case for using Grizzly/Undertow in standalone mode?

File: README.md [hunk 4]
```diff
@@ -385,6 +387,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 
 *Libraries which provide general utility functions.*
 
+* [args4j](http://args4j.kohsuke.org/) - Command line arguments parser
```
Comment (akullpp) [inline README.md]: Ninja change, but okay.

Comment (yegorius) [inline README.md]: Yeah, I have pushed this one after creating pull request, didn't know it will end up here.

File: README.md (hunk 3 above)
Comment (yegorius) [inline README.md]: Undertow is actually a frameork for building high-performance HTTP servers. It has API very similar to Servlets 3.0, but it's just much faster.

Comment (yegorius) [inline README.md]: A valid use case for Grizzly would be the same as for Netty, they actually have very similar purpose. Undertow is made for custom HTTP servers.

File: README.md (hunk 2 above)
Comment (yegorius) [inline README.md]: Because Netty is here and they share similar purpose.

**Pull request merged**